src/init.cpp

```diff
@@ -340,7 +340,6 @@ std::string HelpMessage(HelpMessageMode mode)
 
     string strUsage = HelpMessageGroup(_("Options:"));
     strUsage += HelpMessageOpt("-?", _("This help message"));
-    strUsage += HelpMessageOpt("-addressindex", strprintf(_("bitcore: Maintain a full address index, used to query for the balance, txids and unspent outputs for addresses (default: %u)"), false)); // fAddressIndex (insight_explorer)
```

Comment:
I didn't add a help message for `-insightexplorer` because none of the other experimental options are documented in this part of the help (one-line summaries).